packages/cypress/src/reporter.ts

```diff
@@ -39,6 +39,11 @@ class CypressReporter {
       return [];
     }
 
+    // If the spec doesn't have any tests, we should bail here
```

Comment:
Let's move this up to the top and group it up with the prior bail out. This'll fix SCS-593 too.